ydb/docs/en/core/yql/reference/yql-core/syntax/_includes/create_table.md

```diff
@@ -113,6 +113,17 @@ CREATE TABLE my_table (
 ```
 {% endif %}
 
+{% if feature_temp_tables %}
+{% if feature_olap_tables %}#{%endif%}## Creating a temporary table {#row-additional}
```

Comment:
It's too complicated preprocessing syntax. I'd recommend refactoring this page to avoid that and potential mistakes. For example, it looks like `{#row-additional}` is a duplicate identifier that was used below on L128.